app/src/main/java/com/mk/tasky/agenda/data/local/entity/ReminderEntity.kt

```diff
@@ -0,0 +1,19 @@
+package com.mk.tasky.agenda.data.local.entity
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import java.util.*
+
+@Entity
+data class ReminderEntity(
+    @PrimaryKey
+    val id: String = UUID.randomUUID().toString(),
+    val title: String,
+    val description: String,
+    val remindAt: Long,
+    val day: Int,
+    val month: Int,
+    val year: Int,
+    val hour: Int,
+    val minute: Int
```

Comment:
And what happens if the user switches their timezone? Boom, invalid date 😄 
In a DB, always work with timestamps (as you did with `remindAt`)

Reply:
Fixed!